<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>826</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>110</height>
       </size>
      </property>
      <widget class="QPushButton" name="btnNext">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>70</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Verder</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkLoop">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>70</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Loop</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listFolders">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>0</y>
         <width>941</width>
         <height>61</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QPushButton" name="btnStop">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>70</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPrevious">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>70</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Terug</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRemoveFolder">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>120</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Verwijder map</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblStatus">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Status: klaar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPause">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>70</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Pauze</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAddFolder">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>120</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Voeg map toe</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboSelectType">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Foto's</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Films</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Beide</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="btnStart">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>70</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinPhotoDelay">
       <property name="geometry">
        <rect>
         <x>820</x>
         <y>70</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QFrame" name="mediaFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
